accounting/localizations/mexico.rst

```diff
@@ -16,60 +50,111 @@ and search for Mexico. Then click on *Install*.
    as country when creating your account, the mexican localization will be
    automatically installed.
 
-Accounting Reports
-~~~~~~~~~~~~~~~~~~
+2. Electronic Invoices (CDFI 3.2 and 3.3 format)
+------------------------------------------------
+
+To enable this requirement in Mexico go to configuration in accounting Go in
+:menuselection:`Accounting --> Settings` and enable the option on the image
+with this you will be able to generate the signed invoice (CFDI 3.2 and 3.3)
+and generate the payment complement signed as well (3.3 only) all fully
+integrate with the normal invoicing flow in Odoo.
+
+.. image:: media/mexico02.png
+   :align: center
+
+3. Set you legal information in the company
+-------------------------------------------
+
+First, make sure that your company is configured with the correct data.
+Go in :menuselection:`Settings --> Users --> Companies`
+and enter a valid address and VAT for
+your company. Don’t forget to define a mexican fiscal position on your
+company’s contact.
+
+.. tip::
+   If you want use the Mexican localization on test mode, you can put any known
+   address inside Mexico with all fields for the company address and
+   set the vat to **ACO560518KW7**.
 
-Some specific reports have been added for the Mexican accounting
-localization :
+.. image:: media/mexico03.png
+   :align: center
 
--  DIOT
+4. Set the proper "Fiscal Position" on the partner that represent the company.
+------------------------------------------------------------------------------
 
--  COA
+Go In the same form where you are editing the company save the record in
+order to set this form as a readonly and on readonly view click on the partner
+link, then edit it and set in the *Invoicing* tab the proper Fiscal Information
+(for the **Test Environment** this must be *601 - General de Ley Personas
+Morales*, just search it as a normal Odoo field if you can't see the option).
 
--  Trial Balance
+5. Enabling CFDI Version 3.3
+----------------------------
 
-Electronic Invoices (CDFI format)
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+This steps are only necessary when you will enable the CFDI 3.3 (only available
+for V11.0 and above) if you do not have Version 11.0 or above on your
+SaaS instance please ask for an upgrade sending a ticket to support in
+https://www.odoo.com/help.
```

Comment:
I am not sure the part after "if you do not have..." is relevant. The whole documentation is for Odoo 11 so we hope they have already upgraded :)